### PR TITLE
Slack Detections - EKM

### PR DESCRIPTION
**Pull request by wey-chiang**
### Background

This PR includes new detections for the EKM logs.

New detections include:

* EKM Config Changed
* EKM Slackbot Unenrolled
* EKM Unenrolled

### Changes

* New Detections

### Testing

* Manual Testing
* `make test ci`

### CODE REVIEW

File: slack_rules/slack_ekm_config_changed.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from panther_base_helpers import slack_alert_context
+
+
+def rule(event):
+    # Only alert on the `ekm_logging_config_set` action
+    return event.get("action") == "ekm_logging_config_set"
+
+
+def alert_context(event):
+    # TODO: Add details to the context
```
Comment (k-bailey) [inline slack_rules/slack_ekm_config_changed.py:10]: nit, I suspect this comment can be removed?

File: slack_rules/slack_ekm_slackbot_unenrolled.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Filename: slack_ekm_slackbot_unenrolled.py
+RuleID: Slack.AuditLogs.EKMSlackbotUnenrolled
+DisplayName: Slack EKM Slackbot Unenrolled
+Enabled: true
+LogTypes:
+  - Slack.AuditLogs
+Tags:
+  - Slack
+Severity: High
+Description: Detects when the slack sent notifications are no longer being enrolled in EKM
```
Comment (k-bailey) [inline slack_rules/slack_ekm_slackbot_unenrolled.yml:11]: nit: this wording is a little confusing

File: slack_rules/slack_ekm_config_changed.py (hunk 1 above)
Comment (wey-chiang) [reply slack_rules/slack_ekm_config_changed.py:10]: Once we have this data, this should be updated

**Review by edyesed — APPROVED** [review]
re-approving after working with previous review feedback

**Pull request merged**